app/lists.rb

```diff
@@ -0,0 +1,53 @@
+module Listing
+	extend Discordrb::Commands::CommandContainer
+	
+	wing_list = Array.new
+	#The array that is wing_list is created.
+	
+	command :winglist, description: "Lists those who are requesting wing." do |event|
+	#Simple, return the array's contents. Also, admin lock this.
+		break unless event.user.role?(find_role(event, 'wingcaptain'))
+		#Check to see if they have the "wingcaptain role"
+		event << wing_list.take(wing_list.length)
+		#Why do I have the feeling this will barf data unintelligibly?
+	end
+  
+	command :wingremove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event, target|
+	#Remove someone who is on the wing list. Defo admin lock.
+		break unless event.user.role?(find_role(event, 'wingcaptain'))
+		#Check to see if they have the "wingcaptain role". I am aware this is a double comment.
+		wing_list.delete(target)
+		event.respond << "Removed" + target + "from the queue."
+	end
+	
+	command :wingnuke, description: "Removes everything from the wing list. Useful for after a meetup." do |event|
+		break unless event.user.role?(find_role(event, 'wingcaptain') && find_role(event, 'botadmin'))
```

Comment:
and actually, could do this everywhere.

Or look at the discordrb permissions module thingamabob. Maybe.